### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Queues_ViewMessage_Screenreader: While navigating using Tab key, screenreader is announcing incorrect information about "Expiration Time" box

### ISSUE THREAD

**Issue by Jtith** [microsoft/AzureStorageExplorer#6375]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-nikpap%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847543876&sdata=6wiDneF0tmnVt4CroW43rkfVbB%2BL%2BAAdn0nD7xR5hQM%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**GitHub Tags:**

#A11yMAS; #A11yTCS; #Mac; #DesktopApp; #A11ySev2; #BM_AzureToolsStorageExplorer_Mac_Dec2022; #Azure Tools Storage Explorer; #WCAG1.3.1; #FTP; 
### Environment Details:
Application: Microsoft Azure Storage Explore
version: 1.27.0
### OS Details
Mac version: macOS Ventura 13.1

### Repro Steps:
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Queues" of the selected subscription and expand it.
5. Navigate to testerq and click on it.
6. turn on voiceover.
7.Navigate to View message button and click on it.
8.Navigate using Tab key and observe Expiration time 

### Actual results:
While navigating using Tab key, screenreader is announcing incorrect information about "Expiration Time" box
Similar issue is observed with other information fields.

### Expected results:
Screenreader should announce correct information about "Expiration Time" box

### User Impact:
Screen reader will get confused if incorrect information is being announced. 
### Attachments:

**Comment by Jtith** [comment]
verified in latest environment, Issue is fixed, hence closing the bug
<img width="1440" alt="MicrosoftTeams-image (5)" src="https://user-images.githubusercontent.com/100552367/211789742-6894f974-4e31-4e28-862b-de574a94b059.png">

**Comment by fsteffi** [comment]
GitHubTags:#Closed;
